### PR TITLE
Align TS API Au test with Yahoo internal version.

### PR DESCRIPTION
**Pull request by ywkaras**

### CODE REVIEW

**Comment by ywkaras** [comment]
[approve ci autest]

File: tests/gold_tests/pluginTest/tsapi/tsapi.test.py [hunk 1]
```diff
@@ -26,6 +26,13 @@
 )
 Test.ContinueOnFail = True
 
+plugin_name = "test_tsapi"
+log_file_name = "log.txt"
+
+
+# The test plugin will output test logging to this file.
+Test.Env["OUTPUT_FILE"] = Test.RunDirectory + "/" + log_file_name
```
Comment (bneradt) [inline tests/gold_tests/pluginTest/tsapi/tsapi.test.py:34]: A small thing, but better to use `os.path.join` here as is done elsewhere in the file.

**Review by bneradt — CHANGES_REQUESTED** [review]
Looks good.

**Review by randall — APPROVED** [review]

**Comment by ywkaras** [comment]
> Looks good.

You're still blocking the merge.

**Review by bneradt — APPROVED** [review]

**Pull request merged**